data/elan-42ab.tablet

```diff
@@ -8,13 +7,12 @@
 
 [Device]
 Name=Dell Inc. Inspiron 14 7445 2-in-1
-ModelName=
-DeviceMatch=i2c|04f3|42ab
+ModelName=ELAN3233:00 04F3:42AB
```

Comment:
You can skip this line, the `ModelName` is for devices that have a marketing name and a model name, e.g. a Wacom Intuos Pro may be model number PTK-123. This doesn't really apply here since it's just the firmware nubmers here.